src/datanode/src/version_reporter.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use common_meta::version_reporter::{
+    GreptimeVersionReport, Mode as VersionReporterMode, Reporter, VersionReportTask,
+    VERSION_REPORT_INTERVAL, VERSION_UUID_KEY,
+};
+use object_store::{ErrorKind, ObjectStore};
+use servers::Mode;
+
+use crate::datanode::DatanodeOptions;
+
+fn get_uuid_file_name() -> String {
+    format!("./.{}", VERSION_UUID_KEY)
```

Comment:
If you use OpenDAL, `./` is unnecessary
```suggestion
    format!(".{}", VERSION_UUID_KEY)
```